evmrpc/subscribe.go

```diff
@@ -48,7 +48,15 @@ type QueryBuilder struct {
 	conditions []string
 }
 
-func NewQueryBuilder() *QueryBuilder {
+func NewBlockQueryBuilder() *QueryBuilder {
+	return &QueryBuilder{
+		conditions: []string{
+			"tm.event = 'NewBlock'",
```

Comment:
Based on the Naming convention, will it be better to be "tm.event = 'Block'" ?

Reply:
I believe this is actually a tendermint specific query: https://github.com/tendermint/tendermint/blob/85a584dd2ee3437b117d9f5e894eac70bf09aeef/types/events.go#L19